java/src/main/native/CMakeLists.txt

```diff
@@ -1,22 +1,21 @@
-#=============================================================================
+# =============================================================================
 # Copyright (c) 2019-2021, NVIDIA CORPORATION.
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
 #
-#     http://www.apache.org/licenses/LICENSE-2.0
+# http://www.apache.org/licenses/LICENSE-2.0
 #
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#=============================================================================
+# Unless required by applicable law or agreed to in writing, software distributed under the License
```

Comment:
Yeah that's a fair point. FWIW I just checked rmm (which is already using these cmake formatting tools) and we've gone ahead with the reformatting in those cmake files, so I think we're OK accepting this for now. I don't think we have sufficient control to prevent this.